src/controllers/user-orders-controller.js

```diff
@@ -1,29 +1,56 @@
 import UserOrders from '../models/UserOrders';
+import MenuLogic from '../controllers/menu-controller';
+
+function getOrders(
+  username, startDate = new Date(new Date().setDate(new Date().getDate() - 8)),
+  endDate = new Date(new Date().setDate(new Date().getDate() + 8)),
+) {
+  return UserOrders.findOne({ username }).then(user => ({
+    result: user.days.filter((item) => {
+      console.log(item.date.getTime() > startDate.getTime());
+      return item.date.getTime() > startDate.getTime()
+        && item.date.getTime() < endDate.getTime();
+    }),
+  }));
+}
+
+function validateOrder(order) {
+  return new Promise((resolve, rej) => {
```

Comment:
1. Replace new Promise with return Promise.resolve(...) and return Promise.reject(..)
2. Keep function size under 5 lines